tests/repository_simulator.py

```diff
@@ -139,7 +149,7 @@ def targets(self) -> Targets:
 
     def all_targets(self) -> Iterator[Tuple[str, Targets]]:
         """Yield role name and signed portion of targets one by one."""
-        yield Targets.type, self.md_targets.signed
+        yield "targets", self.md_targets.signed
```

Comment:
What is the reason behind reverting back to strings instead of using the constants?

Reply:
It is related to my mess-up. I fixed it.